### PR TITLE
Symfonize absences index

### PR DESCRIPTION
**Pull request by alexarn**

### CODE REVIEW

File: public/setup/db_data.php [hunk 1]
```diff
@@ -22,7 +22,6 @@
 $sql[]="INSERT INTO `{$dbprefix}acces` (`nom`,`groupe_id`,`groupe`,`page`) VALUES ('Planning par poste - Semaine', 99, '', 'planning/poste/semaine.php');";
 $sql[]="INSERT INTO `{$dbprefix}acces` (`nom`,`groupe_id`,`groupe`,`page`) VALUES ('Aide', 99, '', '/help');";
 $sql[]="INSERT INTO `{$dbprefix}acces` (`nom`,`groupe_id`,`groupe`,`page`) VALUES ('Absences - Index', 100, '', 'absences/index.php');";
```
Comment (jeromecombes) [inline public/setup/db_data.php:24]: absences/index.php can also be deleted

File: templates/absences/add.html.twig [hunk 2]
```diff
@@ -289,7 +289,7 @@
 
             <tr>
               <td colspan='2' style='text-align:center;'>
-                <input type='button' class='ui-button' value='Annuler' onclick="document.location.href='index.php?page=absences/voir.php';" />
+              <input type='button' class='ui-button' value='Annuler' onclick="document.location.href='{{ config('URL') }}/absence';" />
```
Comment (jeromecombes) [inline templates/absences/add.html.twig:292]: Use {{ asset('absence') }}

**Review by jeromecombes — CHANGES_REQUESTED** [review]

**Comment by jeromecombes** [comment]
With APP_DEBUG=1 : 
When I go to /absence, I get "Access denied" with this error :
<pre>Uncaught PHP Exception Twig\Error\RuntimeError: "Variable "visibles_absences" does not exist." at /var/www/html/plb/templates/absences/index.html.twig line 96</pre>

**Comment by jeromecombes** [comment]
> With APP_DEBUG=1 :
> When I go to /absence, I get "Access denied" with this error :
> 
> Uncaught PHP Exception Twig\Error\RuntimeError: "Variable "visibles_absences" does not exist." at /var/www/html/plb/templates/absences/index.html.twig line 96

Fix : 
<pre>
diff --git a/src/Controller/AbsenceController.php b/src/Controller/AbsenceController.php
index c3160fc4..0206a677 100644
--- a/src/Controller/AbsenceController.php
+++ b/src/Controller/AbsenceController.php
@@ -219,11 +219,12 @@ class AbsenceController extends BaseController
                 $visibles_absences[] = $elem;
             }
 
-            $this->templateParams(array(
-                'visibles_absences' => $visibles_absences,
-            ));
         }
 
+        $this->templateParams(array(
+            'visibles_absences' => $visibles_absences,
+        ));
+
         return $this->output('absences/index.html.twig');
     }
</pre>

**Comment by jeromecombes** [comment]
When I click on "Agents supprimés", 
I get this message "Une erreur est survenue lors de la mise à jour de la liste des agents."
and an error 500 on personnel/ajax.updateAgentsList.php?deleted=yes

**Comment by jeromecombes** [comment]
> When I click on "Agents supprimés",
> I get this message "Une erreur est survenue lors de la mise à jour de la liste des agents."
> and an error 500 on personnel/ajax.updateAgentsList.php?deleted=yes

Same error on master.
<pre>2021/05/20 12:42:37 [error] 29337#29337: *1430 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Cannot declare class personnel, because the name is already in use in /var/www/html/plb/public/personnel/class.personnel.php on line 27" while reading response header from upstream, client: 127.0.0.1, server: plb, request: "GET /personnel/ajax.updateAgentsList.php?deleted=yes HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php-fpm.sock:", host: "plb", referrer: "http://plb/absence"</pre>

Fix :
<pre>
diff --git a/public/personnel/ajax.updateAgentsList.php b/public/personnel/ajax.updateAgentsList.php
index 8437aa06..898434ca 100644
--- a/public/personnel/ajax.updateAgentsList.php
+++ b/public/personnel/ajax.updateAgentsList.php
@@ -19,7 +19,7 @@ Appelé en Ajax via la fonction JS updateAgentsList à partir de la page voir.ph
 ini_set('display_errors', 0);
 
 include "../include/config.php";
-include "class.personnel.php";
+include_once "class.personnel.php";
</pre>

**Review by jeromecombes — APPROVED** [review]

**Comment by alexarn** [comment]
Merged on master

**Pull request closed without merging**